op-dispute-mon/metrics/metrics.go

```diff
@@ -333,6 +336,11 @@ func NewMetrics() *Metrics {
 			"result_correctness",
 			"root_agreement",
 		}),
+		latestValidL2Block: factory.NewGauge(prometheus.GaugeOpts{
+			Namespace: Namespace,
+			Name:      "latest_valid_l2_block",
+			Help:      "L2 block number proposed by the most recent game that is expected to be valid",
+		}),
```

Comment:
I think this is actually the same as the existing `latest_proposal` with the label `root_agreement=true`.

Reply:
ah right, sorry I missed that difference.  Latest l2 block number is probably useful as well.

Reply:
Maybe rename to `latest_proposal_block`? Open to your thoughts here, tis not an easy one to name.

Reply:
Yeah agree reporting on invalid games doesn't make sense as they may have a made up block number (you'd basically always report the ridiculously large block number our canary invalid proposals use).  So  `latest_valid_proposal_l2_block` then?